src/sys/ptrace.rs

```diff
@@ -68,9 +73,13 @@ mod ffi {
     }
 }
 
-/// Performs a ptrace request. If the request in question is provided by a specialised function
+/// A low-level wrapper for `ptrace`. If available, the higher-level wrappers should be considered instead.
```

Comment:
This should be a short one-line description with a blank line and then more description below.